src/main/java/org/ow2/proactive/catalog/repository/CatalogObjectRevisionRepositoryImpl.java

```diff
@@ -75,6 +75,14 @@ public List<CatalogObjectRevisionEntity> findDefaultCatalogObjectsOfKindListAndC
                  .collect(Collectors.toList());
     }
 
+    private String toRightSidePredicatePattern(String pattern) {
+        return pattern.contains("%") ? pattern.toLowerCase() : pattern.toLowerCase() + "%";
+    }
+
+    private String toBothSidesPredicatePattern(String pattern) {
+        return pattern.contains("%") ? pattern.toLowerCase() : "%" + pattern + "%";
```

Comment:
Missing `.toLowerCase()` for false case

Reply:
You're right! I forgot it. It will be fixed. Thanks!